flink-dstl/flink-dstl-dfs/src/test/java/org/apache/flink/changelog/fs/StateChangeFsUploaderTest.java

```diff
@@ -19,18 +19,32 @@
 package org.apache.flink.changelog.fs;
 
 import org.apache.flink.api.common.JobID;
+import org.apache.flink.core.fs.FSDataOutputStream;
+import org.apache.flink.core.fs.FileSystem;
 import org.apache.flink.core.fs.Path;
+import org.apache.flink.core.fs.local.LocalDataOutputStream;
+import org.apache.flink.core.fs.local.LocalFileSystem;
 
+import org.junit.Rule;
 import org.junit.jupiter.api.Test;
+import org.junit.rules.TemporaryFolder;
 
+import java.io.File;
 import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.apache.flink.runtime.metrics.groups.UnregisteredMetricGroups.createUnregisteredTaskManagerJobMetricGroup;
+import static org.apache.flink.util.Preconditions.checkNotNull;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /** {@link StateChangeFsUploader} test. */
 class StateChangeFsUploaderTest {
```

Comment:
@masteryhx Thanks for contribution, would you want to migrate this test class to Junit5 and AssertJ, and rewrite testCompress in that way.

Reply:
Thanks for the suggestion. Modified.

Reply:
For assertion, I suggest using AssertJ instead of Junit's, and you can find the reason here: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing